src/inversion_scripts/point_sources.py

```diff
@@ -60,17 +63,53 @@ def get_point_source_coordinates(config):
 
     # then we read point sources from external datasources
     plumes = []
+    got_plumes = False
+
     if "PointSourceDatasets" in config.keys():
-        if "SRON" in config["PointSourceDatasets"]:
-            print("Fetching plumes from SRON database...")
-            plumes = SRON_plumes(config)
-        else:
+
+        # List of datasources
+        ps_datasets = copy(config["PointSourceDatasets"])
+        ## Have to treat SRON separately for now
+        #if 'SRON' in ps_datasets:
+        #    ps_datasets.remove('SRON')
+
+        #    print("Fetching plumes from SRON database...")
+        #    plumes += SRON_plumes(config)
+        #    got_plumes = True
```

Comment:
remove comment?

Reply:
Removed. Since SRON is now incorporated into plumes.py along with CarbonMapper and IMEO, I also removed old functions from this file that are no longer used.